test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Index.Encoded.html

```diff
@@ -39,35 +39,13 @@
         </a>
     </div>
     <div>
-        <a href="HtmlEncode[[/Customer/Customer#fragment]]">Customer Area Customer Index</a>
+        <a href="HtmlEncode[[/Customer/HtmlGeneration_Customer#fragment]]">Customer Area Customer Index</a>
```

Comment:
There is no `CustomerController` but there is a `HtmlGeneration_CustomerController`